dms/models/storage.py

```diff
@@ -85,6 +85,19 @@ class Storage(models.Model):
     )
 
     model_ids = fields.Many2many("ir.model", string="Linked Models")
+    inherit_access_from_parent_record = fields.Boolean(
+        string="Inherit access from parent record",
+        default=False,
+        help="Indicate if directories and files access work only with "
+        "related model access (for example, if some directories are related "
+        "with any sale, only users with read access to these sale can acess)",
+    )
+    mail_post_autofollow = fields.Boolean(
+        string="Mail post autoflow",
```

Comment:
Remember the string is for humans 😋:

```suggestion
        string="Create files from message attachments",
```